PowerPointLabs/PowerPointLabs/CropToShape.cs

```diff
@@ -243,10 +243,10 @@ private static void CreateRotatedFillInBackground(PowerPoint.Shape shape, Bitmap
             }
         }
 
-        private static void TakeScreenshotProxy(PowerPoint.ShapeRange shapeRange)

```

Comment:
this magnifyRatio is passed in every method, what do you think if turn it into a state/field/member of class CropToShape